content/engine/context/working-with-contexts.md

```diff
@@ -22,46 +28,43 @@ To follow the examples in this guide, you'll need:
 
 Run `docker context` to verify that your Docker client supports contexts.
 
-You will also need one of the following:
-
-- Docker Swarm cluster
-- Single-engine Docker node
-- Kubernetes cluster
-
 ## The anatomy of a context
 
 A context is a combination of several properties. These include:
 
-- Name
+- Name and description
 - Endpoint configuration
 - TLS info
-- Orchestrator
 
 The easiest way to see what a context looks like is to view the **default** context.
 
 ```console
 $ docker context ls
-NAME          DESCRIPTION     DOCKER ENDPOINT                KUBERNETES ENDPOINT      ORCHESTRATOR
-default *     Current...      unix:///var/run/docker.sock                             swarm
+NAME        DESCRIPTION                               DOCKER ENDPOINT               ERROR
+default *                                             tcp://docker:2375
+myctx                                                 unix:///var/run/docker.sock
 ```
 
-This shows a single context called "default". It's configured to talk to a Swarm cluster through the local `/var/run/docker.sock` Unix socket. It has no Kubernetes endpoint configured.
+This shows a single context called "default".
```

Comment:
This is actually listing two: `default` and `myctx`.  Could edit this description or the `ls` output.